package.json

```diff
@@ -1578,50 +1700,65 @@
           "when": "view == jobLog && viewItem =~ /^project.*/",
           "group": "1_viewFile@0"
         },
+        {
+          "command": "vscode-ibmi-projectexplorer.jobLog.clearJobLogs",
+          "when": "view == jobLog && viewItem =~ /^project.*/",
+          "group": "inline@1"
+        },
         {
           "command": "vscode-ibmi-projectexplorer.jobLog.clearJobLogs",
           "when": "view == jobLog && viewItem =~ /^project.*/",
           "group": "2_modification@0"
         },
         {
           "command": "vscode-ibmi-projectexplorer.jobLog.copy",
-          "when": "view == jobLog && viewItem =~ /^command.*/",
+          "when": "view == jobLog && viewItem =~ /^(command|commandRepresentation).*/",
           "group": "inline@0"
         },
         {
           "command": "vscode-ibmi-projectexplorer.jobLog.copy",
-          "when": "view == jobLog && viewItem =~ /^command.*/",
-          "group": "1_copy"
+          "when": "view == jobLog && viewItem =~ /^(command|commandRepresentation).*/",
```

Comment:
what is a `commandRepresentation`

Reply:
When you and Irfan made the changes for the job log, `commandRepresentation` was the context value set for the following tree item:

![image](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/32170854/aeddd41c-b067-4c78-a8c3-72097854571a)

Reply:
I think we should update this tree item from `commandRepresentation` -> `command` and the parent one from `command` -> `object` which is more accurate. I can make this change in a separate PR since I will do a bunch of renaming.